paddle/majel/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(MAJEL_CXX_FILES place.cpp)
```

Comment:
PADDLE includes both CPU and GPU versions.  `Majel` is built with GPU only if it equipped one. 

If CMake detects the host machine equipped GPUs, it will invoke `cuda_add_library` to generate the library, vice visa.

**NOTE: ** The suffix of unit test files should strictly limit to `*.cpp`, because it doesn't contain any CUDA syntaxes and semantics.